browser/forms.py

```diff
@@ -329,3 +329,9 @@ def save(self, user):
                     'changed_by': user,
                 })
         return new_person
+
+
+class InvestigationForm(forms.ModelForm):
```

Comment:
this should be called "InvestigatorForm" according to its model class

Reply:
Changed the name to "InvestigatorForm" from "InvestigationForm"